grails-app/conf/application.yml

```diff
@@ -20,6 +20,11 @@ environments:
             url: jdbc:mysql://localhost/streama
             username: root
             password:
+    development-no-mysql:
+            dataSource:
+                dbCreate: update
+                driverClassName:  'org.h2.Driver'
```

Comment:
Interesting choice, although for easier debugging I actually use mysql locally. Maybe each contributor should set this as desired without pushing?